ci/driver.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash --login
+#
+#####################################################################################
+#
+# Script description: Top level driver script for checking PR
+#                     ready for CI regresssion testing
+#
+# Abstract:
+#
+# This script uses GitHub CL to check for Pull Requests with {machine}-CI tags on the
+# development branch for the global-workflow repo.  It then stages tests directors per
+# PR number and calls run_ci.sh to perform a clone and full build from $(HOMEgfs)/sorc
+# of the PR. It then is ready to run a suite of regression tests with various
+# configurations with run_tests.py.
+#######################################################################################
+
+#################################################################
+# TODO using static build for GitHub CLI until fixed in HPC-Stack
+#################################################################
+GH=/home/Terry.McGuinness/bin/gh
+repo_url=${repo_url:-"https://github.com/global-workflow.git"}
+
+################################################################
+# Setup the reletive paths to scripts and PS4 for better logging 
+################################################################
+set -eux
+pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+scriptname=$(basename "${BASH_SOURCE[0]}")
+start_time=$(date +%s)
+start_time_human=$(date -d"@${start_time}" -u)
+echo "Begin ${scriptname} at ${start_time_human}"
+export PS4='+ $(basename ${BASH_SOURCE})[${LINENO}]'
+
+host=$(hostname) || true
+
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -t <target> -h"
+  echo
+  echo "  -t  target/machine script is running on    DEFAULT: ${host}"
+  echo "  -h  display this message and quit"
+  echo
+  exit 1
+}
+
+#########################################################################
+#  Set up runtime environment varibles for accounts on supproted machines
+#########################################################################
+TARGET=$(hostname) || true
+export TARGET
+while getopts "t:h" opt; do
+  case ${opt} in
+    t)
+      TARGET=${OPTARG}
+      ;;
+    h|\?|:)
+      usage
+      ;;
+    *)
+      echo "Unrecognized option" 
+      usage
+      exit
+  esac
+done
+
+case ${TARGET} in
+  hera | orion)
+    echo "Running Automated Testing on ${TARGET}"
+    source "${pwd}/${TARGET}.sh"
+    ;;
+  *)
+    echo "Unsupported platform. Exiting with error."
+    exit 1
+    ;;
+esac
+
+#########################################################################
+# pull on the repo and get list of open PRs with tags {machine}-CI
+#########################################################################
+mkdir -p "${GFS_CI_ROOT}/repo"
+cd "${GFS_CI_ROOT}/repo"
+if [[ ! -d "${GFS_CI_ROOT}/repo/global-workflow" ]]; then
+ git clone "${repo_url}"
+fi
+cd "${GFS_CI_ROOT}/repo/global-workflow"
+git pull
+
+CI_LABEL="${GFS_CI_HOST}"
```

Comment:
```suggestion
CI_HOST="${GFS_CI_HOST}"
```
I think `CI_HOST` makes more sense since the values of this variable are `Orion` or `Hera`.
The label is `${CI_HOST}-${CI_STATUS}` where `CI_STATUS` can be `CI|Running|Passed|Failed`